fuzz/Cargo.toml

```diff
@@ -20,7 +20,7 @@ block = { path = "../block" }
 devices = { path = "../devices" }
 epoll = "4.3.3"
 hypervisor = { path = "../hypervisor", features = ["mshv_emulator"] }
-libc = "0.2.155"
```

Comment:
All very good - just maybe a bit more explanation in this commit about why you made these changes.

Reply:
Not needed anymore; see https://github.com/cloud-hypervisor/cloud-hypervisor/compare/623eeeccf97fe9844ae30187ac0484798f51aaaf..bfbd00211e121a28ede2b2f18cc552e3af438507